stack/python/stack.py

```diff
@@ -0,0 +1,63 @@
+class Stack:
+    """
+    Class for operations related to stack.
+    """
+
+    def __init__(self):
+
+        self.stack = [] # Initialise empty stack
+
```

Comment:
The code does not comply to PEP8.

*Origin: PEP8Bear, Section: `all.autopep8`.*

The issue can be fixed by applying the following patch:

```diff
--- a/tmp/tmple4xxr40/stack/python/stack.py
+++ b/tmp/tmple4xxr40/stack/python/stack.py
@@ -5,7 +5,7 @@
 
     def __init__(self):
 
-        self.stack = [] # Initialise empty stack
+        self.stack = []  # Initialise empty stack
 
     def push(self, dataval):
         """
```